### PR TITLE
Add note about ports range format

### PR DESCRIPTION
**Pull request by dhmhd**
* Add note for LocalPort and RemotePort options that ports range format can be used for specifying a continuous sequence of ports in one single ACL rule;
* Add usage example for this feature.

### CODE REVIEW

**Comment by e0i** [comment]
Reopening to run checks

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012r2-ps/hyper-v/Add-VMNetworkAdapterExtendedAcl.md [hunk 1]
```diff
@@ -210,7 +215,7 @@ Accept wildcard characters: False
 ```
 
 ### -LocalPort
-Specifies the local port for the ACL.
+Specifies the local port for the ACL. Also a ports range format can be used (i.e. "49152-49182", for example).
```
Comment (get-itips) [inline docset/winserver2012r2-ps/hyper-v/Add-VMNetworkAdapterExtendedAcl.md:218]:
```suggestion
Specifies the local port for the ACL. A port range format can be used also (i.e. "49152-49182", for example).
```

File: docset/winserver2012r2-ps/hyper-v/Add-VMNetworkAdapterExtendedAcl.md [hunk 2]
```diff
@@ -298,7 +303,7 @@ Accept wildcard characters: False
 ```
 
 ### -RemotePort
-Specifies the remote port for the ACL.
+Specifies the remote port for the ACL. Also a ports range format can be used (i.e. "49152-49182", for example).
```
Comment (get-itips) [inline docset/winserver2012r2-ps/hyper-v/Add-VMNetworkAdapterExtendedAcl.md:306]:
```suggestion
Specifies the remote port for the ACL. A port range format can be used also  (i.e. "49152-49182", for example).
```

**Review by get-itips — COMMENTED** [review]
Thanks for the contribution

File: docset/winserver2012r2-ps/hyper-v/Add-VMNetworkAdapterExtendedAcl.md (hunk 2 above)
Comment (dariomws) [inline docset/winserver2012r2-ps/hyper-v/Add-VMNetworkAdapterExtendedAcl.md:306]:
```suggestion
Specifies the local port for the ACL. A port range format can also be used (i.e. "49152-49182", for example).
```

File: docset/winserver2012r2-ps/hyper-v/Add-VMNetworkAdapterExtendedAcl.md (hunk 1 above)
Comment (dariomws) [inline docset/winserver2012r2-ps/hyper-v/Add-VMNetworkAdapterExtendedAcl.md:218]:
```suggestion
Specifies the local port for the ACL. A port range format can also be used (i.e. "49152-49182", for example).
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Comment by dariomws** [comment]
The same information can also be added to W2016 article.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Sorry Andres, have to go with Dario on this one.

**Review by tseward — APPROVED** [review]
Approved pending dariomws' changes.

**Comment by e0i** [comment]
@dhmhd This PR is good to go once the suggested changes have been applied. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiburd** [comment]
close > reopen

**Pull request merged**